### DISCUSSION TITLE
Decrease polling interval

### DISCUSSION THREAD

**Discussion by jurgenskrause** [StephanJoubert/home_assistant_solarman#369] [Q&A]
I would like to get faster updates of the load on the inverter to trigger switching my geyser on and off when there is excess solar available. How do I change the polling interval to it's minimum?

I have a Deye 5kw inverter that is working well with HA through this plugin

**Comment by PV-Joe** [comment]
Go to const.py and set the interval here:
MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=15)

**Reply by jurgenskrause** [reply]
Great stuff, thank you.

**Reply by jurgenskrause** [reply]
Hi @PV-Joe 
Thanks for the suggestion, I have changed the value, but it still seems to update only every 30s

**Comment by PV-Joe** [comment]
That is what happens to me as well.
I changed it to 30 sec and restarted HA.
It worked for a while but updated every 60 sec.

Another change to 20 sec and a HA restart set it and is working for weeks now.

Did you restart HA?
